certs/templates/user_data.tpl

```diff
@@ -1,13 +1,20 @@
 #cloud-config
 runcmd:
+  - export INSTANCE_ID=`curl http://169.254.169.254/latest/meta-data/instance-id`
   - echo "OPENVPN_CERT_SOURCE=s3://${replace(s3_bucket,"/(/)+$/","")}/${replace(s3_bucket_prefix,"/^(/)+|(/)+$/","")}" > /etc/openvpn/get-openvpn-certs.env
+  - if [ -n "${vpc_dns_ip}" ]; then echo "push \"dhcp-option DNS ${vpc_dns_ip}\"" >> /etc/openvpn/server.conf;fi
+  - echo 'crl-verify /etc/openvpn/keys/crl.pem' >> /etc/openvpn/server.conf
```

Comment:
Probably outside the scope here, but we may want to eventually add the support for t2 unlimited to the user data (@kjonick1 improvement this week)

Reply:
@pickgr yeah though that's falling a little bit more into the custom user_data capabilities that i'm trying to get into the newer module being built. will keep that in mind.